### PR TITLE
feat: add button to switch the documentation

### LINKED ISSUE #1089: Bug: add ability to switch between open-sauced and docs
### Describe the bug

The [docs](https://docs.opensauced.pizza) is missing a "home" button, both the logo and the "docs" link going back to the docs endpoint

### To Reproduce

1. Go to https://docs.opensauced.pizza
2. Never go back

### Expected behavior

Should be able to visit docs and opensauced.pizza without opening a new tab

### Screenshots

Screenshot 2021-07-15 at 13 49 16

### Desktop

- OS:
- Browser:
- Version:

### Smartphone

- Device:
- OS:
- Browser:
- Version:

### PR DESCRIPTION
**Pull request by daptheHuman**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [X] ✨ Feature
- [ ] 🐛 Bug Fix
- [X] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🔖 Release
- [ ] 🚩 Other

## 

-  Description

This PR addresses the issues:

- Adding feature to switching the [docs](https://docs.opensauced.pizza/) on top right corner

## Related Tickets & Documents

Fixes #1089 

## Mobile & Desktop Screenshots/Recordings

image

## Added tests?

- [X] 👍 yes
but on the docs it need to tested out, I don't know how to edit the [docs](https://docs.opensauced.pizza/) to add home button
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [X] 📓 docs
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?
N/A

## [optional] What gif best describes this PR or how it makes you feel?
image

### CODE REVIEW

**Comment by filiptronicek** [comment]
Hi, @daptheHuman, thanks for this contribution! As @0-vortex mentioned in the original issue, could you also make this work vice-a-versa? In other words, make it possible to get from the docs to opensauced.pizza?

**Comment by daptheHuman** [comment]
> Hi, @daptheHuman, thanks for this contribution! As @0-vortex mentioned in the original issue, could you also make this work vice-a-versa? In other words, make it possible to get from the docs to opensauced.pizza?

yeah i really want to do that, but i think it is not allowed to change the docs or i just don't know🤔

**Review by 0-vortex — APPROVED** [review]
LGTM

File: src/components/Nav.js [hunk 1]
```diff
@@ -81,6 +81,7 @@ function RightSide({user}) {
   return (
     <FloatRight>
       <SpaceBetween>
+        <SubtleLink href="https://docs.opensauced.pizza/" className="nav-link" rel="noreferrer">Docs </SubtleLink>
```
[the remark this answers is not in the document]
Comment (daptheHuman) [reply src/components/Nav.js]: #https://github.com/open-sauced/open-sauced/pull/1095/commits/9102f36bbd64e148861a1f56703144a79c1b28cc

  already change in here

[the remark this answers is not in the document]
Comment (0-vortex) [reply src/components/Nav.js]: I know, resolved the conversation, all done just wait :D

**Review by bdougie — APPROVED** [review]
LGTM! Thanks for contributing.

**Pull request merged**